src/wallet.ts

```diff
@@ -339,11 +349,13 @@ export class Wallet {
     })
 
     // @ts-ignore: method does not exist on Contract type
-    await contract.nft_batch_transfer(
-      { token_ids: tokenIds },
-      MAX_GAS,
-      ONE_YOCTO
-    )
+    await contract.nft_batch_transfer({
+      meta: 'nft_batch_transfer',
```

Comment:
See below, we just silently override user input?

Reply:
Good point, will remove these.